src/utils/theme/theme.js

```diff
@@ -0,0 +1,62 @@
+const RELATIVE_UNIT_BASE = 16;
+const MAX_WIDTH = '1255px';
+
+const breakpoints = {
+  desktop: 960,
+  tablet: 768,
+  phone: 540,
+};
+
+const generateBreakpoint = (width, cssMarkup) => `
+  @media (max-width: ${width / RELATIVE_UNIT_BASE}em) {
```

Comment:
I belive we can have some problems with this calculation on the future, please move it to: 

```suggestion
  @media (max-width: ${width}px) {
```

Reply:
Done.